workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/SpatialOSEntity.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Improbable.Worker;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public struct SpatialOSEntity
+    {
+        private readonly EntityManager entityManager;
+        private readonly Entity entity;
+        public readonly EntityId SpatialEntityId;
+        public readonly int UnityEntityIndex;
```

Comment:
no need to store that separately. Only the entity is enough, especially because the index is not a unique identifier

Reply:
This is to expose it to the users of the class, I guess it could be a method instead. Bit weird when the Spatial id is a field though.

Reply:
The index won't give the users the full identifier of the entity. I just fixed a bug for this and I want to make sure we don't run into this again.
If you want to expose it to the user, make the entity field public.